mappings/net/minecraft/world/gen/feature/ConfiguredFeatures.mapping

```diff
@@ -1,5 +1,13 @@
-CLASS net/minecraft/class_5464 net/minecraft/TerrainFeatures
-	CLASS class_5467 Features
+CLASS net/minecraft/class_5464 net/minecraft/world/gen/feature/ConfiguredFeatures
+	METHOD method_30590 register (Ljava/lang/String;Lnet/minecraft/class_2975;)Lnet/minecraft/class_2975;
+	CLASS class_5465 Configs
+	CLASS class_5466 Decorators
+		FIELD field_26157 FIRE Lnet/minecraft/class_3243;
+		FIELD field_26158 HEIGHTMAP Lnet/minecraft/class_3243;
+		FIELD field_26159 TOP_SOLID_HEIGHTMAP Lnet/minecraft/class_3243;
+		FIELD field_26160 HEIGHTMAP_WORLD_SURFACE Lnet/minecraft/class_3243;
+		FIELD field_26161 HEIGHTMAP_SPREAD_DOUBLE Lnet/minecraft/class_3243;
```

Comment:
hmm, inconsistent names. should all be XXX_HEIGHTMAP like i think?

Reply:
humm possibly, I just took these from the auto generated names:

![](https://s.modm.us/java_2020-07-10_13-21-44.png)